sources/node2/KalturaClientBase.js

```diff
@@ -166,6 +166,9 @@ class ClientBase extends kaltura.RequestData {
 		if (config.getLogger() instanceof ILogger) {
 			this.shouldLog = true;
 		}
+		if(config.sessionId) {

```

Comment:
I would expect to see `sessionId` on request-level, not in client-level, because we could create a client once for requests with different sessionIds. (I understand that in CCProxy we create a client for each request, but anyway).
I suggest adding method in `RequestBuilder` class
```
sessionId(id) {
    this.sessionId = id;
    return this;
}
```

and in CCProxy extend signature of `eventNotificationActionDispatch` with sessionId and update
```
kaltura.services.eventNotificationAction.dispatch(scope)
    .sessionId(sessionId)   
    .completion(....
```